modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad/src/main/java/com/liferay/dynamic/data/mapping/uad/display/DDMFormInstanceRecordUADDisplay.java

```diff
@@ -97,9 +97,6 @@ public String getEditURL(
 			).put(
 				portletNamespace.concat("mvcPath"),
 				new String[] {"/admin/edit_form_instance_record.jsp"}
-			).put(
-				portletNamespace.concat("readOnly"),
-				new String[] {Boolean.FALSE.toString()}
```

Comment:
@marcosapmf, just to make more explicit to the next pull request reviewer, I believe that would be better move this modification to a separate commit and explain it through the commit message, can you please do this?